contracts/fund/policies/utils/AddressListPolicyMixin.sol

```diff
@@ -13,7 +13,7 @@ abstract contract AddressListPolicyMixin {
 
     event AddressesRemoved(address policyManager, address[] items);
 
-    mapping (address => EnumerableSet.AddressSet) policyManagerToList;
+    mapping (address => EnumerableSet.AddressSet) private policyManagerToList;
```

Comment:
This is fine, but it could arguably be `internal` since it's an abstract contract that is only inherited (i.e., a child may have some use for it).